app/products/calls/actions/calls.ts

```diff
@@ -263,3 +275,97 @@ export const setSpeakerphoneOn = (speakerphoneOn: boolean) => {
     InCallManager.setSpeakerphoneOn(speakerphoneOn);
     setSpeakerPhone(speakerphoneOn);
 };
+
+export const endCallAlert = async (serverUrl: string, intl: IntlShape, channelId: string) => {
```

Comment:
This function is mostly new (needs deeper review than the rest of the code, which is mostly a port)